app/controllers/training_and_support_controller.rb

```diff
@@ -10,34 +10,38 @@ def show
       raise Pundit::NotAuthorizedError, "This page is only available to CIP participants"
     end
 
+    current_user.participant_profile.update!(guidance_seen: true)
     @provider_name_key = @provider.name.downcase.gsub(/ /, "_")
   end
 
   def update
-    @participant = current_user.participant_profile
+    @guidance_speedbump_form = GuidanceSpeedbumpForm.new(guidance_question_params)
 
-    if params[@participant.model_name.param_key].blank?
-      @participant.errors.add :viewed_guidance, "Select if you would like a brief summary of the programme"
+    if @guidance_speedbump_form.view_guidance_option.blank?
+      @guidance_speedbump_form.errors.add :view_guidance_option, "Select if you would like a brief summary of the programme"
       render :guidance_question and return
     end
 
-    @participant.update!(show_guidance_speedbump: false)
+    current_user.participant_profile.update!(guidance_seen: true)
 
-    if view_guidance?
+    if view_guidance?(@guidance_speedbump_form)
       redirect_to training_and_support_path
     else
       redirect_to cip_path(current_user.core_induction_programme)
     end
   end
 
   def guidance_question
-    @participant = current_user.participant_profile
+    @guidance_speedbump_form = GuidanceSpeedbumpForm.new
   end
 
 private
 
-  def view_guidance?
-    participant = current_user.participant_profile
-    params[participant.model_name.param_key][:show_guidance_speedbump] == "view guidance"
+  def view_guidance?(guidance_speedbump_form)
+    guidance_speedbump_form.view_guidance_option == "view_guidance"
+  end
```

Comment:
Maybe this wants to be a method on the form? so in your controller you would write `@guidance_speedbump_form.view_guidance?`, and in your form
```
  def view_guidance?
    view_guidance_option == "view_guidance"
  end
```

Reply:
Yeah, makes sense, will move it.